src/partials/hero.html

```diff
@@ -1,3 +1,28 @@
 <section id="hero" class="section hero-section">
-  <div class="container">Hero</div>
+  <div class="container hero-content">
+      <div class="hero-tittle">
+        <h1>Discover the joy of learning <span class="english-green">English</span> with us!</h1>
+      </div>
+      <button class="btn cta-button">Leave an application</button>
+      
+      <div class="round-button"></div>
```

Comment:
1. that should not be <div> with background image! this is an svg.
2. if position absolute here, it is better to do from the botton and right, not top left